docs/content/community/radius-artifacts.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "Artifacts on Radius"
+linkTitle: "Artifacts on Radius"
+description: "Information about press, blog posts, presentation and other artifacts on Radius"
+weight: 400
+---
+
+## Microsoft Blog posts
```

Comment:
Let's make this more generic. We don't want to make it just Microsoft-specific

```suggestion
## Blog posts
```

Reply:
good catch!